src/components/modules/banners/contributions/variantA/ContributionsBannerVariantA.tsx

```diff
@@ -0,0 +1,182 @@
+        margin-top: 8px;
+    `,
+    ctaContainer: css`
+        margin-bottom: 16px;
+        display: flex;
+        justify-content: flex-end;
+    `,
+
+    tabletAndDesktop: css`
+        display: none;
+        ${between.tablet.and.leftCol} {
+            display: block;
+        }
+    `,
+    leftCol: css`
+        display: none;
+        ${between.leftCol.and.wide} {
+            display: block;
+        }
+    `,
+    wide: css`
+        display: none;
+        ${from.wide} {
+            display: block;
+        }
+    `,
+};
+
+const columnCounts = {
+    tablet: 12,
+    desktop: 12,
+    leftCol: 14,
+    wide: 16,
+};
+
+const ContributionsBannerVariantA: React.FC<ContributionsBannerProps> = ({
+    onContributeClick,
+    onCloseClick,
+    cleanHighlightedText,
+    cleanMessageText,
+    cleanMobileMessageText,
+    cleanHeading,
+    ctaUrl,
+    ctaText,
+}: ContributionsBannerProps) => {
+    const BodyAndHeading = () => (
+        <div css={variantAStyles.bodyAndHeading}>
+            <div css={variantAStyles.heading}>{cleanHeading}</div>
+            <div css={variantAStyles.body}>
+                <div css={variantAStyles.rule} />
+                <div css={[styles.copy, variantAStyles.copy]}>
+                    {cleanMessageText}
+                    {cleanHighlightedText && (
+                        <>
+                            {' '}
+                            <span css={styles.highlightedText}>{cleanHighlightedText}</span>
+                        </>
+                    )}
+                </div>
+            </div>
+        </div>
+    );
+
+    const buttons = (
+        <div css={variantAStyles.buttonsContainer}>
+            <ContributionsBannerCloseButton onCloseClick={onCloseClick} />
+            <div css={variantAStyles.ctaContainer}>
+                <ContributionsBannerCta
+                    onContributeClick={onContributeClick}
+                    ctaText={ctaText}
+                    ctaUrl={ctaUrl}
+                    stacked={true}
+                />
+            </div>
+        </div>
+    );
+
+    return (
+        <div css={styles.bannerContainer}>
+            <ContributionsBannerMobile
+                onCloseClick={onCloseClick}
+                onContributeClick={onContributeClick}
+                heading={cleanHeading}
+                messageText={cleanMobileMessageText || cleanMessageText}
+                highlightedText={cleanHighlightedText}
+                ctaUrl={ctaUrl}
+                ctaText={ctaText}
+            />
+
+            <Container cssOverrides={variantAStyles.columnsContainer}>
+                <div css={variantAStyles.tabletAndDesktop}>
+                    <Columns>
+                        <Column width={8 / columnCounts.tablet}>
+                            <BodyAndHeading />
+                        </Column>
+                        <Column width={4 / columnCounts.tablet}>{buttons}</Column>
+                    </Columns>
+                </div>
```

Comment:
I doesn't really make much of a difference, but there is a `Hide` component that we could use instead of these custom styles. I think it would be
```jsx
<Hide below="tablet" above="desktop">
   ...
</Hide>
```

Reply:
Unfortunately with `Hide` it's either `below` or `above`.
Simon is looking at making it easier to do this stuff, but for now we have to use media queries in this situation

Reply:
Oh that's a real shame. So you'd have to do

```jsx
<Hide below="tablet">
  <Hide above="desktop">
    ...
  </Hide>
</Hide>
```

Yeah that's not ideal! Good feedback for him though 👍